src/app/work-item/work-item-list/work-item-list.component.html

```diff
@@ -1,17 +1,17 @@
 <app-header></app-header>
 <div class="container-fluid screen-min-ht">
 	<div class="list-group list-view-pf list-view-pf-view">
-		 <div>
+		 <div id="quickAddWorkItemWrapper">
 			 <div class="list-group-item supress-hover" (click)="addWorkItem()">
 				<work-item-quick-add (close)="close($event)"></work-item-quick-add>
 			</div>
 		</div>
-		<div *ngFor="let workItem of workItems">
-			<div class="list-group-item" (click)="onSelect(workItem)"
+		<div *ngFor="let workItem of workItems" id="workItemList">
```

Comment:
I guess this will give multiple element with same id. In cases like this we should always use an index number. For example, workItemList_1, workItemList_2 etc. This should be done programmatically, i.e  {{'workItemList' + '_' + index}}